gadget/gadget.go

```diff
@@ -339,6 +343,8 @@ func InfoFromGadgetYaml(gadgetYaml []byte, model Model) (*Info, error) {
 	var bootloadersFound int
 	knownFsLabelsPerVolume := make(map[string]map[string]bool, len(gi.Volumes))
 	for name, v := range gi.Volumes {
+		// set the VolumeName for the volume
+		v.VolumeName = name
 		if err := validateVolume(name, v, knownFsLabelsPerVolume); err != nil {
```

Comment:
this can drop the name parameter now
```suggestion
		if err := validateVolume(v, knownFsLabelsPerVolume); err != nil {
```

Reply:
done